src/textual/app.py

```diff
@@ -562,7 +562,20 @@ def __init__(
         """Number of lines to scroll in the Y direction with wheel or trackpad."""
 
         self._installed_screens: dict[str, Screen | Callable[[], Screen]] = {}
+        for var_name, dct in (("SCREENS", self.SCREENS), ("MODES", self.MODES)):
```

Comment:
We have "no abbreviates" rule in Textual. So `var_name` and `dct` would have to be something descriptive.

The constructor may not be the best place for this. As you are checking classvars, it seems redundant to do this work every time the App is instantiated.

Suggest `__init_subclass__` is the right place for this check. That way it would run just once, at import time...